components/ciphertool/src/main/java/org/wso2/ciphertool/utils/Utils.java

```diff
@@ -366,5 +366,25 @@ public static String doEncryption(Cipher cipher, String plainTextPwd) {
         return encodedValue;
     }
 
+    /**
+     * Checks if a string is blank
+     *
+     * @param str       string value
+     */
+
+    public static boolean isBlank(String str) {

```

Comment:
Why do we need to define our own isBlank method. We can use org.apache.commons.lang3.StringUtils.isBlank(..) or isEmpty(..) as relevant.

Reply:
Added the suggested changes